modules/users/client/views/profile/profile-view.client.view.html

```diff
@@ -251,15 +251,7 @@ <h1>Don't panic!</h1>
                   ng-if="app.appSettings.referencesEnabled"
                 >
                   <a ui-sref="profile.references.list" role="tab">
-                    <span aria-hidden="true">References</span>
-                    <span
-                      class="badge"
-                      aria-hidden="true"
-                      ng-bind="profileCtrl.referencesCount || 0"
-                    ></span>
-                    <span class="sr-only">
-                      {{profileCtrl.referencesCount || 0}} References
-                    </span>
```

Comment:
Simplified the tab for now by taking the count away; I'll add this as a todo to implement.